ng-file-upload/index.d.ts

```diff
@@ -41,7 +41,11 @@ declare module 'angular' {
              * Validate error name: minDuration
              * @type {(number|string)}
              */
-            ngfMinDuration: number | string;
+            ngfMinDuration?: number | string;
+            /**
+             * Validate error name: minSize
+             * @type {(number|string)}
```

Comment:
Why is the @type doc-comment-field necessary when the type is encoded in the declaration below?